chordspy/chords_wifi.py

```
@@ -1,17 +1,53 @@
+"""
+CHORDS WiFi Data Acquisition Script: This script provides a WebSocket client for connecting to and receiving data from
+a CHORDS-compatible WiFi device. It handles connection management, data reception, and basic data validation.
+"""
+
+# Importing necessary libraries
 import time
 import sys
 import websocket
 import socket
 from scipy.signal import butter, filtfilt
```

Ruff (0.11.9)

`scipy.signal.butter` imported but unused

Remove unused import

(F401)